### PR TITLE
Update RunSettingsArguments.md

### PR DESCRIPTION
**Pull request by Faizan2304**
Add information that `Runsettings arguments` should be specified at the end of the command line

### CODE REVIEW

File: docs/RunSettingsArguments.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
-
 # Passing runsettings arguments through commandline
 You are here because you are looking for syntax & details to pass runsettings configurations to either `vstest.console.exe` or `dotnet test` through commandline.
```
Comment (harshjain2) [inline docs/RunSettingsArguments.md]: Suggestion : & -> and

File: docs/RunSettingsArguments.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 
-`runSettings arguments` are used to add/update specific `runsettings configurations`. The updated `runsettings configurations` will be available to `TestPlatform` and `test extensions` (E.g. test adapter, etc.) through runsettings.
+`RunSettings arguments` are used to add/update specific `runsettings configurations`. The updated `runsettings configurations` will be available to `TestPlatform` and `test extensions` (E.g. test adapter, etc.) through runsettings.
 
-`runSettings arguments` are specified as name-value pair of the form `[name]=[value]` after `-- `. Note the space after --. Use a space to separate multiple `[name]=[value]`.
+`RunSettings arguments` are specified as name-value pair of the form `[name]=[value]` after `-- `. Note the space after --. Use a space to separate multiple `[name]=[value]`. All the arguments after `--` will be treated as `RunSettings arguments`, means `RunSettings arguments` should be at the end of the command line.
```
Comment (harshjain2) [inline docs/RunSettingsArguments.md]: Suggestion : List down all the conditions as bullet points.

File: docs/RunSettingsArguments.md (hunk 1 above)
Comment (Faizan2304) [inline docs/RunSettingsArguments.md]: Fixed

File: docs/RunSettingsArguments.md (hunk 2 above)
Comment (Faizan2304) [inline docs/RunSettingsArguments.md]: Fixed

**Pull request merged**